### PR TITLE
Update tg.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**

### CODE REVIEW

File: streams/tg.m3u [hunk 1]
```diff
@@ -17,3 +17,9 @@ https://hls.newworldtv.com/nw-sport-2/video/live.m3u8
 https://pvqyb34odz24-hls-live.5centscdn.com/doc2/tv.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="",NW Sport 3 (576p)
 https://hls.newworldtv.com/nw-sport-3/video/live.m3u8
+#EXTINF:-1 tvg-id="",NW Transat  (576p) [Not 24/7]
+https://hls.newworldtv.com/nw-transnat/video/live.m3u8
+#EXTINF:-1 tvg-id="",NW Economie (576p)
+https://hls.newworldtv.com/nw-economie/video/live.m3u8
+#EXTINF:-1 tvg-id="",NW Sport 4 (576p)
+https://hls.newworldtv.com/nw-sport-4/video/live.m3u8
```
Comment (freearhey) [inline streams/tg.m3u:25]: Offline.

image

File: streams/tg.m3u [hunk 2]
```diff
@@ -17,3 +17,9 @@ https://hls.newworldtv.com/nw-sport-2/video/live.m3u8
 https://pvqyb34odz24-hls-live.5centscdn.com/doc2/tv.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="",NW Sport 3 (576p)
 https://hls.newworldtv.com/nw-sport-3/video/live.m3u8
+#EXTINF:-1 tvg-id="",NW Transat  (576p) [Not 24/7]
+https://hls.newworldtv.com/nw-transnat/video/live.m3u8
+#EXTINF:-1 tvg-id="",NW Economie (576p)
+https://hls.newworldtv.com/nw-economie/video/live.m3u8
```
Comment (freearhey) [inline streams/tg.m3u:23]: Offline.

image

**Review by freearhey — DISMISSED** [review]

File: streams/tg.m3u (hunk 2 above)
Comment (BellezaEmporium) [reply streams/tg.m3u:23]: Playlist gets in, but no live data found afterwards.

File: streams/tg.m3u [hunk 3]
```diff
@@ -17,3 +17,9 @@ https://hls.newworldtv.com/nw-sport-2/video/live.m3u8
 https://pvqyb34odz24-hls-live.5centscdn.com/doc2/tv.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="",NW Sport 3 (576p)
 https://hls.newworldtv.com/nw-sport-3/video/live.m3u8
+#EXTINF:-1 tvg-id="",NW Transat  (576p) [Not 24/7]
+https://hls.newworldtv.com/nw-transnat/video/live.m3u8
```
Comment (BellezaEmporium) [inline streams/tg.m3u:21]: Playlist gets in, but no live data found afterwards.

File: streams/tg.m3u (hunk 1 above)
Comment (BellezaEmporium) [reply streams/tg.m3u:25]: works

**Review by BellezaEmporium — COMMENTED** [review]
snip

File: streams/tg.m3u (hunk 3 above)
Comment (freearhey) [inline streams/tg.m3u:21]:
```suggestion
```

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**